skin/langs/translation-en.json

```
@@ -1,6 +1,9 @@
 {
   "third": "Third",
   "page.actions.create.label": "Create a new document",
+  "page.creation.menu.alert.content": "The page {pageName} already exists, use another name.",
```

Maybe just me but I feel like explaining the page already exists is enough.
```suggestion
  "page.creation.menu.alert.content": "The page {pageName} already exists.",
```